From ca9e86428801eef6c9712f85b7e53417c2b68a62 Mon Sep 17 00:00:00 2001
From: Sabri Ünal <yakushabb@gmail.com>
Date: Tue, 5 Nov 2024 14:47:58 +0300
Subject: [PATCH] Fix icon name

---
 lib/solaar/ui/about.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/solaar/ui/about.py b/lib/solaar/ui/about.py
index e10c7f9..689cfc7 100644
--- a/lib/solaar/ui/about.py
+++ b/lib/solaar/ui/about.py
@@ -36,8 +36,8 @@ def _create():
     about.set_program_name(NAME)
     about.set_version(__version__)
     about.set_comments(_("Manages Logitech receivers,\nkeyboards, mice, and tablets."))
-    about.set_icon_name(NAME.lower())
-    about.set_logo_icon_name(NAME.lower())
+    about.set_icon_name("io.github.pwr_solaar.solaar")
+    about.set_logo_icon_name("io.github.pwr_solaar.solaar")
 
     about.set_copyright("© 2012-2023 Daniel Pavel and contributors to the Solaar project")
     about.set_license_type(Gtk.License.GPL_2_0)
--
libgit2 1.7.2

